### PR TITLE
add Pisen_WPR003N

### PR DESCRIPTION
**Pull request by libinanshu**
Specifications:
- QCA9341 SoC, 16 MB nor flash, 64 MB DDR2 RAM
- wifi 2.4Ghz 300Mbps
- 1x Ethernet LAN 10/100, 1x Ethernet WAN 10/100
- 1x PWR  LED
- 1x USB
- Reset buttons

### CODE REVIEW

**Review by DragonBluep — CHANGES_REQUESTED** [review]
Please signoff your commit and force push it again. Maybe your commit message can be more detailed. Just use
```
git commit --amend --signoff
git push ---force-with-lease origin master
```
Thank you!

**Comment by csharper2005** [comment]
It would be better to rename commit title according to naming convention. for example, ramips: add support for D-Link DAP-1325-A1. Please add flash instruction.

File: target/linux/ath79/dts/ar9341_pisen_wpr003n.dts [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+		spi-max-frequency = <25000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			uboot: partition@0 {
+				label = "u-boot";
+				reg = <0x000000 0x020000>;
```
Comment (csharper2005) [inline target/linux/ath79/dts/ar9341_pisen_wpr003n.dts:58]: Unnecessary zeros can be deleted.

**Comment by adschm** [comment]
https://openwrt.org/docs/guide-developer/frequent-pr-mistakes#do_not_multi-post_or_re-post_prs

**Pull request closed without merging**